ts/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -31,6 +32,8 @@ import { SharedStorageService, SharedStorageConfig } from './sharedStorage';
 import { NFSSharedStorageService } from './shared_storages/nfsStorageService'
 import { AzureBlobSharedStorageService } from './shared_storages/azureblobStorageService'
 import { TrialDetail } from './trial';
+import { LinuxCommands } from "../remote_machine/extends/linuxCommands";
```

Comment:
trialDispatcher.ts is a common service, should not import common commands from remote_machine folder.

Reply:
Removed